packages/theme/src/document-head-contents/element-theme-script.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { STORAGE_THEME_MANUAL_KEY, ThemeName, VOID_FN } from '../globals'
+import { getTopLevelDomain } from '@lidofinance/utils'
+import {COOKIE_THEME_MANUAL_KEY, COOKIES_THEME_EXPIRES_DAYS} from "../../../cookie-theme-toggler/src/constants";
+
+/**
+ *  this FN should be hermetic and has zero external items - it is inlined via .toString()
+ * */
+/*#__PURE__*/
```

Comment:
Why does this PURE mean? This function is referring document.cookie, it isn't pure 🤔

Reply:
this is special syntax for minifiers and transpilers.
basically it allows function to be inlined, but here it is used as a (not fully working but) canary test that should break things if this function will use external logic or variables